webpack.config.js

```diff
@@ -0,0 +1,27 @@
+/*global __dirname, require*/
```

Comment:
I can't remember whether this does anything but I added it because it looked vaguely relevant